packages/components/src/components/card/card.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import React, { HTMLProps } from 'react';
+
+type TGeneralCardComponentsProps = HTMLProps<HTMLDivElement>;
+
+type TCardProps = HTMLProps<HTMLDivElement> & {
+    header: React.ReactNode;
+    renderHeader: () => React.ReactNode;
+    content: React.ReactNode;
+    renderContent: () => React.ReactNode;
+    footer: React.ReactNode;
+    renderFooter: () => React.ReactNode;
```

Comment:
@jim-deriv Not sure why we have render props as well if we are not passing any data in the callback 🤔

Reply:
@farzin-fs, I'm wondering the same thing, but that flexibility was provided by our predecessors and yet we already had the `header`, `content` and `footer` props. Maybe there is a reason behind it.

Reply:
@jim-deriv I don’t know man, These TS migration cards don’t make much sense to me in general 😅
We are just changing files from `jsx` to `tsx` and replacing `propTypes` with `types` but not fixing the implementation itself so TS doesn’t really help us even after everything is in TS.

Let’s move it forward 🙇🏻🌹

Reply:
@farzin-fs, Yeah. I actually think we should create separate cards to refactor such